### PR TITLE
Feature/reduce requests

### PR DESCRIPTION
**Pull request by fbinz**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -4,32 +4,139 @@ build-backend = "flit_core.buildapi"
 
 [project]
 name = "django-dynamic-admin-forms"
-authors = [{name = "Fabian Binz", email = "fabian.binz@ambient.digital"}]
+authors = [
+    {'name' = 'Ambient Digital', 'email' = 'hello@ambient.digital'},
+    {'name' = 'Ambient Digital', 'email' = 'fabian.binz@ambient.digital'},
```
Comment (GitRon) [inline pyproject.toml]: fabian binz gehört da hin (im meta.py müsste das drin stehen)

**Comment by GitRon** [comment]
@fbinz Linting has an issue as well, it seems

File: pyproject.toml (hunk 1 above)
Comment (fbinz) [reply pyproject.toml]: Done :+1:

File: .ambient-package-update/metadata.py [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+$ cd testproj
+$ python manage.py runserver 0.0.0.0:8000 &  # start server
+$ python manage.py loaddata fixtures/fixtures-dev.json
+$ cd ../e2e
+$ yarn install  # or npm install (only needed first time)
+$ yarn cypress  # or npm run cypress
+```
+
+## Changelog
+- 0.2.0: Reduce amount of requests, move to ambient-package-update
```
Comment (GitRon) [inline .ambient-package-update/metadata.py]: i have a neat solution in the toolbox package - might want to have a look there 🙂

File: LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (GitRon) [inline LICENSE.md:1]: es gibt noch eine alte `LICENSE`, die müsste dan nweg

File: docs/index.rst [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+The package is published at pypi under the following link: `https://pypi.org/project/django-dynamic-admin-forms/ <https://pypi.org/project/django-dynamic-admin-forms/>`_
+
+This README is mostly a placeholder, if at some time this package needs more documentation.
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Contents:
+
+   features/changelog.md
```
Comment (GitRon) [inline docs/index.rst:7]: why not add the readme here as well? then we have proper docs 🙂

**Review by GitRon — APPROVED** [review]

**Pull request merged**